### PR TITLE
Update style to adhere to google style guide

### PR DESCRIPTION
**Pull request by alexander871015**
Update style for the server

### CODE REVIEW

File: hologram_dashboard/server/request_handler/request_handler.h [hunk 1]
```diff
@@ -1,81 +1,57 @@
...
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
 */
+
 #include <pistache/http.h>
 #include <pistache/router.h>
 #include <pistache/endpoint.h>
 
 using namespace Pistache;
+namespace wireless_android_play_analytics {
```
Comment (sarahchen-google) [inline hologram_dashboard/server/request_handler/request_handler.h:22]: blank line between line 21 and 22.

File: hologram_dashboard/server/request_handler/request_handler.cc [hunk 2]
```diff
@@ -18,44 +18,45 @@
 #include <sstream>
 using namespace Pistache;
 using namespace std;
```
Comment (sarahchen-google) [inline hologram_dashboard/server/request_handler/request_handler.cc:20]: Blank line after 16, 18, 19,20

File: hologram_dashboard/server/request_handler/request_handler.cc [hunk 3]
```diff
@@ -18,44 +18,45 @@
...
+namespace wireless_android_play_analytics {
```
Comment (sarahchen-google) [inline hologram_dashboard/server/request_handler/request_handler.cc:21]: Update copyright header?

File: hologram_dashboard/server/request_handler/request_handler.cc [hunk 4]
```diff
@@ -18,44 +18,45 @@
...
+void RequestHandler::GetLastRefreshed(const Rest::Request& request, 
     Http::ResponseWriter response) {
-    // Information within the request is unnecessary
+    // Information within the request is unnecessary.
     UNUSED(request);
     response.send(Http::Code::Ok, "Wed May 19 15:46:11 2020");
 }
 
-void RequestHandler::getDashboard(const Rest::Request& request, 
+void RequestHandler::GetDashboard(const Rest::Request& request,
```
Comment (sarahchen-google) [inline hologram_dashboard/server/request_handler/request_handler.cc:59]: Add TODO here to explain why you are using hard-coded values.

File: hologram_dashboard/server/request_handler/request_handler.cc [hunk 5]
```diff
@@ -1,62 +1,69 @@
...
 */
+
 #include "request_handler.h"
+
 #include <vector>
 #include <sstream>
+
 using namespace Pistache;
+
 using namespace std;
```
Comment (sarahchen-google) [inline hologram_dashboard/server/request_handler/request_handler.cc:24]: Don't use

using namespace Pistache;

using namespace std;

You can include these header in variables, like std::XXX

File: hologram_dashboard/server/request_handler/request_handler.h [hunk 6]
```diff
@@ -1,81 +1,58 @@
...
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
 */
+
 #include <pistache/http.h>
 #include <pistache/router.h>
 #include <pistache/endpoint.h>
 
 using namespace Pistache;
```
Comment (sarahchen-google) [inline hologram_dashboard/server/request_handler/request_handler.h:21]: Same above.

File: hologram_dashboard/server/request_handler/request_handler.cc [hunk 7]
```diff
@@ -1,62 +1,69 @@
...
-    server.setHandler(router.handler());
+    // Binds the router with correct request methods.
+    Routes::Get(router_, "/:system", 
+        Routes::bind(&RequestHandler::GetDashboard, this));
+    Routes::Get(router_, "/:system/:source/:date", 
+        Routes::bind(&RequestHandler::SendCommand, this));
+    Routes::Get(router_, "/", 
+        Routes::bind(&RequestHandler::GetLastRefreshed, this));
+
+    server_.setHandler(router_.handler());
```
Comment (emilyqu) [inline hologram_dashboard/server/request_handler/request_handler.cc:50]: Prefer to only keep apis for RequestHandler in this class, and move server related init and registering path to functions to server.cc.

File: hologram_dashboard/server/request_handler/request_handler.cc [hunk 8]
```diff
@@ -1,62 +1,69 @@
...
+
 using namespace std;
 
+namespace wireless_android_play_analytics {
+
 RequestHandler::RequestHandler()
-    :server(Address(Ipv4::any(), Port(8000))) {
-        init();
+    :server_(Address(Ipv4::any(), Port(8000))) {
+        Init();
```
Comment (emilyqu) [inline hologram_dashboard/server/request_handler/request_handler.cc:30]: nit: make the port a flag rather than hard-coded number.

Comment (emilyqu) [reply hologram_dashboard/server/request_handler/request_handler.cc:30]: But according to my previous comment, I feel server related init could be in server.cc.

File: hologram_dashboard/server/request_handler/request_handler.cc (hunk 7 above)
Comment (emilyqu) [reply hologram_dashboard/server/request_handler/request_handler.cc:50]: And add tests for RequestHandler.

File: hologram_dashboard/server/request_handler/request_handler.h [hunk 9]
```diff
@@ -1,81 +1,58 @@
...
-        /*
-        * Requires: request is a valid request of form /{system}/{source}/{date}
-        * Modifies: response
-        * Effect: Constructs a command to backfill a specified pipe and send said
-        *         command to the user
-        */
-        void sendCommand(const Rest::Request& request, 
+        // Constructs a command to backfill a specified pipe and send said 
+        // command to the user.
+        void SendCommand(const Rest::Request& request,
```
Comment (emilyqu) [inline hologram_dashboard/server/request_handler/request_handler.h:43]: nit: PopulateBackfillCommand

File: hologram_dashboard/server/request_handler/request_handler.cc (hunk 8 above)
Comment (emilyqu) [reply hologram_dashboard/server/request_handler/request_handler.cc:26]: As discussed offline, will resolve this in a separate pr.

**Pull request merged**